package.json

```diff
@@ -3,8 +3,8 @@
   "description": "A library for interacting with themoviedb.org API using Promises",
   "version": "3.4.1",
   "scripts": {
-    "test": "npm run build && ./node_modules/.bin/mocha ./tests/*.js --reporter spec",
-    "build": "rm -rf ./dist && ./node_modules/.bin/tsc",
+    "test": "npm run build && mocha ./tests/*.js --reporter spec",
```

Comment:
Locally installed NPM binaries (found in ```node_modules/.bin```) are automatically being made available in the ```PATH``` variable during script execution, therefore there is no need to reference their exact location manually ([Reference](https://michael-kuehnel.de/tooling/2018/03/22/helpers-and-tips-for-npm-run-scripts.html)).

Reply:
Sorry to be so long in reviewing this. That article references this situation for `npx`.

I must have a global `tsc` installed, because when I run `build` without specifying `./node_modules/.bin/tsc`, it fails. I had to remove my global installation before `build` worked without specifying the relative path.